tests/config/format-test.js

```diff
@@ -278,6 +291,11 @@ function runTest({
   let formatResult = mainParserFormatResult;
   let formatTestTitle = "format";
 
+  // [prettierx]: disable test files by parser/options
+  if (isDisabled(filename, parser, formatOptions, false)) {
+    return;
+  }
```

Comment:
Thanks @adalinesimonian, but I don't think we need to disable any tests at this point in the module. I started refactoring to rename the function to `skipRoundTripErrorTest` and make the changes above a little more consistent with `unstableTests`.

Reply:
Agreed, better to keep things simple.